packages/esm-ward-app/src/routes.json

```diff
@@ -31,16 +31,37 @@
       "slot": "ward-dashboard-slot"
     },
     {
-    "component": "wardPatientActionButtonExtension",
-    "name": "ward-patient-action-button",
-    "slot": "action-menu-ward-patient-items-slot"
-  }],
+      "component": "wardPatientActionButtonExtension",
+      "name": "ward-patient-action-button",
+      "slot": "action-menu-ward-patient-items-slot"
+    },
+    {
+      "name": "ward-patient-notes-form",
+      "component": "wardPatientNotesFormExtension",
+      "slot": "ward-patient-notes-workspace-slot",
+      "order": 1
+    },
+    {
+      "component": "wardPatientNotesActionButtonExtension",
+      "name": "ward-in-patient-notes-action-button",
+      "slot": "action-menu-ward-patient-items-slot"
+    }
+  ],
   "workspaces": [
     {
-      "name":"admission-requests-workspace",
+      "name": "admission-requests-workspace",
       "component": "admissionRequestWorkspace",
-      "title":"admissionRequests",
-      "type":"admission-requests"
+      "title": "admissionRequests",
+      "type": "admission-requests"
+    },
+    {
+      "name": "ward-patient-notes-workspace",
+      "component": "wardPatientNotesWorkspace",
+      "type": "ward-patient-notes",
+      "title": "In-patient notes",
+      "sidebarFamily": "ward-patient",
+      "width": "wider",
```

Comment:
Why are we using `width` as wider?
It should be `narrow`, which is the default value.

Only clinical forms have `wider` width and all other workspaces (except ward patient workspace) use the `narrow` width for the workspaces.